tests/e2e/specs/in-game/oneOffs.spec.js

```diff
@@ -42,40 +42,268 @@ describe('Untargeted One-Offs', () => {
+        secondCard: Card.EIGHT_OF_HEARTS,
+      });
+      // Player plays five
+      cy.playOneOffAndResolveAsPlayer(Card.FIVE_OF_SPADES);
+  
+      // Assert game state
+      assertGameState(0, {
+        // Player is P0
+        p0Hand: [Card.ACE_OF_CLUBS, Card.FIVE_OF_HEARTS, Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS],
+        p0Points: [Card.TEN_OF_SPADES, Card.ACE_OF_SPADES],
+        p0FaceCards: [Card.KING_OF_SPADES],
+        // Opponent is P1
+        p1Hand: [Card.ACE_OF_HEARTS],
+        p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
+        p1FaceCards: [Card.KING_OF_HEARTS],
+        scrap: [Card.FIVE_OF_SPADES],
+      });
+      // Attempt to plays five out of turn
+      cy.get('[data-player-hand-card=5-2]').click(); // five of hearts
+      playOutOfTurn('oneOff');
+    }); // End five one-off
+
+    it('Plays a 5 to draw the last two cards in the deck', () => {
+      // Setup: player has one card in hand and only top & second card are in deck
+      cy.loadGameFixture(0, {
+        // Player is P0
+        p0Hand: [Card.FIVE_OF_CLUBS],
+        p0Points: [],
+        p0FaceCards: [],
+        // Opponent is P1
+        p1Hand: [],
+        p1Points: [],
+        p1FaceCards: [],
+        // Deck
+        topCard: Card.THREE_OF_CLUBS,
+        secondCard: Card.EIGHT_OF_HEARTS,
+        deck: [],
+      });
+      cy.get('#deck').should('contain', '(2)');
+
+      // Player plays and resolves a 5
+      cy.playOneOffAndResolveAsPlayer(Card.FIVE_OF_CLUBS);
+      assertGameState(0, {
+        // Player is P0
+        p0Hand: [Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS],
+        p0Points: [],
+        p0FaceCards: [],
+        // Opponent is P1
+        p1Hand: [],
+        p1Points: [],
+        p1FaceCards: [],
+        scrap: [Card.FIVE_OF_CLUBS],
+      });
+      // Deck should now be empty
+      cy.get('#deck').should('contain', '(0)').should('contain', 'PASS');
     });
-    // Player plays five
-    cy.playOneOffAndResolveAsPlayer(Card.FIVE_OF_SPADES);
 
-    // Assert game state
-    assertGameState(0, {
-      // Player is P0
-      p0Hand: [Card.ACE_OF_CLUBS, Card.FIVE_OF_HEARTS, Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS],
-      p0Points: [Card.TEN_OF_SPADES, Card.ACE_OF_SPADES],
-      p0FaceCards: [Card.KING_OF_SPADES],
-      // Opponent is P1
-      p1Hand: [Card.ACE_OF_HEARTS],
-      p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
-      p1FaceCards: [Card.KING_OF_HEARTS],
-      scrap: [Card.FIVE_OF_SPADES],
+    it('Plays a 5 to draw two cards when there are 3 cards in deck', () => {
+      // Setup: there are three cards in the deck and player has a 5
+      cy.loadGameFixture(0, {
+        p0Hand: [Card.FIVE_OF_CLUBS],
+        p0Points: [],
+        p0FaceCards: [],
+        p1Hand: [Card.TWO_OF_CLUBS],
+        p1Points: [],
+        p1FaceCards: [],
+        topCard: Card.THREE_OF_CLUBS,
+        secondCard: Card.EIGHT_OF_HEARTS,
+        deck: [Card.ACE_OF_DIAMONDS]
```

Comment:
My initial implementation of 'delete the deck but leave some cards in it' was a revision to `cy.deleteDeck()` so that you'd first load the game fixture without this line then call
```
cy.deleteDeck([Card.ACE_OF_DIAMONDS]);
```

I think it's much more legible folding this into the `cy.loadGameFixture()` call like this instead, in part because it feels weird to have the `cy.deleteDeck()` call take an argument of things to _not_ delete, and in part because ultimately deleting the cards in the deck is still just setup for the test so it makes sense as test fixture.

Ultimately, this actually deprecates the entire need for the `cy.deleteDeck()` command and corresponding endpoint, as any tests that call it can instead just update their `loadFixture()` call to set the `deck: []`. To mitigate scope creep, I'll refrain from updating all those other tests in this update and write up an issue to remove the deleteDeck command + endpoint and replace its use with revised `loadFixture()` calls

Reply:
#421